<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import Data</class>
 <widget class="QMainWindow" name="Import Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>통계 처리</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>861</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>580</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt; 이전</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>완료</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>861</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>통계 처리</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="compareTab_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>881</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab2_2">
     <attribute name="title">
      <string>데이터비교</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
         </font>
        </property>
        <property name="title">
         <string>4분위수 비교</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>401</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="beforePlot"/>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>100</y>
           <width>411</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="afterPlot"/>
        </widget>
        <widget class="QComboBox" name="AllPart">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>131</width>
           <height>46</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
          </font>
         </property>
         <item>
          <property name="text">
           <string>ALL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IQR</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="function">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>120</width>
           <height>46</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
          </font>
         </property>
         <item>
          <property name="text">
           <string>총합</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>평균</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="columns">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>120</width>
           <height>46</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="group">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>20</y>
           <width>120</width>
           <height>46</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="runButton">
         <property name="geometry">
          <rect>
           <x>740</x>
           <y>20</y>
           <width>101</width>
           <height>46</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
          </font>
         </property>
         <property name="text">
          <string>실행</string>
         </property>
        </widget>
        <widget class="QPushButton" name="addButton">
         <property name="geometry">
          <rect>
           <x>619</x>
           <y>20</y>
           <width>101</width>
           <height>46</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>맑은 고딕</family>
          </font>
         </property>
         <property name="text">
          <string>추가</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab1_2">
     <attribute name="title">
      <string>Levels</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableWidget" name="LevelTable">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>범위</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>대체방법</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="font">
           <font>
            <family>맑은 고딕</family>
           </font>
          </property>
          <property name="text">
           <string>삭제</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
